README.md

```diff
@@ -16,29 +16,40 @@ Deis v2 and Helm are changing quickly. Your feedback and participation are more
 
 Hacking on Deis v2 is a bit rough right now--please help us fix the bugs and improve the installation process.
 
-First, add this Chart repo to Helm to install the "deis" chart:
+***First*** you need to have a working Kubernetes cluster on Google Container Engine (Deis, of course, works anywhere Kubernetes can be installed, but the installation instructions below are tailored to GKE, should also work on AWS)
+
+- add this Chart repo to Helm to install the "deis" chart:
```

Comment:
Let's keep it formally to paragraphs rather than bullet points. It reads smoother, IMO